### PR TITLE
Took webgl-check out from the loop

### PR DESCRIPTION
**Pull request by hanbyul-here**
- Checking webgl (creating canvas with webgl context) was inside of loop, creating webgl many times. took it out from the loop

### CODE REVIEW

File: src/js/components/tangram.js [hunk 1]
```diff
@@ -26,31 +26,31 @@ var TangramLayer = L.Class.extend({
   },
 
   addTo: function (map) {
-    if (this._hasWebGL()) {
-      if (typeof Tangram === 'undefined') {
+    if (typeof Tangram === 'undefined') {
+      if (this._hasWebGL()) {
         return window.setTimeout(this.addTo.bind(this, map), 100);
```
Comment (rfriberg) [inline src/js/components/tangram.js]: It looks like `_hasWebGL()` will still be called multiple times due to the setTimeout loop.  We really need to find a better solution to that.  Is there an event that fires once `Tangram` is available?

Comment (rfriberg) [inline src/js/components/tangram.js]: If not, maybe use something like this? https://gist.github.com/ThomasBurleson/5674241#file-scriptinjection-js

**Comment by hanbyul-here** [comment]
@rfriberg yess good point. made haswebgl as a property of tangram component

File: src/js/components/tangram.js [hunk 2]
```diff
@@ -70,6 +71,7 @@ var TangramLayer = L.Class.extend({
     throw new URIError('The script ' + oError.target.src + ' is not accessible.');
   },
   _hasWebGL: function () {
+    console.log('haswebgl?')
```
Comment (rfriberg) [inline src/js/components/tangram.js]: can you remove this, please?

**Comment by rfriberg** [comment]
Thanks for updating. Just one comment inline.

File: src/js/components/tangram.js (hunk 2 above)
Comment (hanbyul-here) [inline src/js/components/tangram.js]: facepalm 😢  thanks

**Pull request merged**